### ISSUE TITLE
Data interface for querying is not available anymore in 0.8.5

### ISSUE THREAD

**Issue by bbinet** [influxdata/influxdb#1069]
I've just upgraded InfluxDB from 0.8.2 to 0.8.5, and when I enter InfluxDB administration ui on port 8083, there is no item available in the "Interfaces" menu, so I cannot access the data interface to run my InfluxDB queries.
In InfluxDB v0.8.2, I could go to the InfluxDB query data interface by selecting "Default" in the "Interfaces" menu.

I tried to investigate this issue with the firefox developer tools, and when I log in, the "/interfaces" url is requested but returns error 400 "Bad request", as you can see in the following screenshot:
![influxdb_screenshot](https://cloud.githubusercontent.com/assets/76281/4827018/a917172c-5f75-11e4-945f-57c1cec0807b.png)

If I try to open this "/interfaces" url in a new tab, I get: "open interfaces: no such file or directory"

**Comment by mkarquette** [comment]
@JulienChampseix using 0.8.8 I wondering if I am not having the same issue? I cannot access the web interface from a remote server or any other influx ports remotely? Was trying to pus graphite data into port 2003. The web interface works great local but all off the ports for influxdb seem closed? I wrote some testing code to try and connect to server ports. Port 80 is open however none of the ports are open for influx. Sorry if I should of posted to another thread.

**Comment by pauldix** [comment]
@mkarquette that sounds like an issue with your setup. Make sure you have the ports open.

**Comment by mkarquette** [comment]
Paul I think you are correct I a missing a concept here about influxdb. 
all of the other ports appear open as iptables off etc. 
also here is a little test script I ran from another server to check. 

def checkPort (hostIn, portIn):
    sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
    result = sock.connect_ex((hostIn,portIn))
    if result == 0:
       print "Port is open"
    else:
       print "Port is not open"

checkPort("172.16.200.145", 8083)
checkPort("172.16.200.145", 8086)
checkPort("172.16.200.145", 80)
checkPort("172.16.200.145", 443)
output is
Port is not open
Port is not open
Port is open
Port is open

Process finished with exit code 0
---
here is info from config.toml
bind-address = "172.16.200.145"
[admin]
port   = 8083 
[api]
port     = 8086
------
note if I shutdown influx. with 
server influxdb stop
then 
I run a test server I wrote on the same server as influxdb using the same ports the ports accept connections fine. 

so it must be something I am missing in the config.toml setup? Note --  I had to config the etc/hosts and hostname on this server so this server doesn't show up in dns. Also the host name will have a local domain ext.
